Doesn't work with node >= 7

During `installDeps.sh` there are multiple deprecations:
```
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated graceful-fs@2.0.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@1.0.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
```
the `graceful-fs` package causes etherpad to be incompatible with node >= 7.0 and causes etherpad to crash with
```
 module.js:474
     throw err;
     ^
 Error: Cannot find module 'internal/fs'
     at Function.Module._resolveFilename (module.js:472:15)
     at Function.Module._load (module.js:420:25)
     at Module.require (module.js:500:17)
     at require (internal/module.js:20:19)
     at evalmachine.<anonymous>:17:20
     at Object.<anonymous> (/opt/etherpad/src/node_modules/npm/node_modules/graceful-fs/fs.js:11:1)
     at Module._compile (module.js:573:32)
     at Object.Module._extensions..js (module.js:582:10)
     at Module.load (module.js:490:32)
     at tryModuleLoad (module.js:449:12)
```

--- a/src/package.json
+++ b/src/package.json
@@ -28,7 +28,7 @@
                       "log4js"                  : "0.6.35",
                       "cheerio"                 : "0.20.0",
                       "async-stacktrace"        : "0.0.2",
-                      "npm"                     : "2.7.6",
+                      "npm"                     : "4.0.2",
                       "ejs"                     : "2.4.1",
                       "graceful-fs"             : "4.1.3",
                       "slide"                   : "1.1.6",

--- a/src/static/js/pluginfw/plugins.js
+++ b/src/static/js/pluginfw/plugins.js
@@ -117,13 +117,14 @@ exports.getPackages = function (cb) {
           delete packages[name].parent;
         }
       
-        if (deps[name].dependencies !== undefined) flatten(deps[name].dependencies);
+        // I don't think we need recursion
+        //if (deps[name].dependencies !== undefined) flatten(deps[name].dependencies);
       });
     }
   
     var tmp = {};
     tmp[data.name] = data;
-    flatten(tmp);
+    flatten(tmp[undefined].dependencies);
     cb(null, packages);
   });
 };

